docs/comparison/nbdb.md

```diff
@@ -22,53 +22,44 @@ kernelspec:
 Requirements:
 
 ```sh
-pip install scikit-learn sklearn-evaluation ploomber jupysql
+pip install sklearn-evaluation ploomber-engine
 ```
 
 ```{code-cell} ipython3
 from pathlib import Path
 
-# to train models in parallel
-from ploomber import DAG
-from ploomber.tasks import NotebookRunner
+import jupytext
+
 from ploomber.products import File
-from ploomber.executors import Parallel, Serial
+from ploomber_engine.tracking import track_execution
```

Comment:
looks like we're not using FIle or track_execution, let's delete them

Reply:
Sounds good